cves/CVE-2011-3923.yml

```diff
@@ -182,8 +177,11 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: |
+      Since the HTTP paramaters were not being properly escaped, OLGN functiona calls
```

Comment:
Small spelling error on "functions"

Reply:
Thanks! I will fix this in the next commit.